addons/mrp_subcontracting/views/subcontracting_portal_templates.xml

```diff
@@ -13,8 +13,9 @@
 
     <template id="portal_my_home_menu_production" name="Portal layout : production menu entries" inherit_id="portal.portal_breadcrumbs" priority="25">
         <xpath expr="//ol[hasclass('o_portal_submenu')]" position="inside">
-            <li t-if="page_name == 'production' or production" t-attf-class="breadcrumb-item #{'active' if not pickings else ''}">
-                <t>Productions</t>
+            <li t-if="title" t-attf-class="breadcrumb-item #{'active' if breadcrumbs_url else ''}">
```

Comment:
![image](https://github.com/odoo/odoo/assets/57942166/1f6ced0f-3963-42e0-a6d6-a2bc573a649f)
![image](https://github.com/odoo/odoo/assets/57942166/a1d32e33-914a-4bd8-85d8-3d0e9dca9714)
Like on the purchase portal,  I think it's better to have `Home/ Production / XXX` and make `Production` clickable to go back to the list. What do you think?

Reply:
It's indeed better, it was the planned behavior of my changes but I think I managed to revert some of them(without me noticing it) and it was not working anymore.
Should be better now. 👌